nextroute/schema/input.go

```diff
@@ -7,129 +7,217 @@ import (
 
 // Input is the default input schema for nextroute.
 type Input struct {
-	Options        any              `json:"options,omitempty"`
-	CustomData     any              `json:"custom_data,omitempty"`
-	Defaults       *Defaults        `json:"defaults,omitempty"`
-	StopGroups     *[][]string      `json:"stop_groups,omitempty"`
-	DurationMatrix *[][]float64     `json:"duration_matrix,omitempty"`
-	DistanceMatrix *[][]float64     `json:"distance_matrix,omitempty"`
+	// Options arbitrary options.
+	Options any `json:"options,omitempty"`
+	// CustomData arbitrary custom data.
+	CustomData any `json:"custom_data,omitempty"`
+	// Defaults default properties for vehicles and stops.
+	Defaults *Defaults `json:"defaults,omitempty"`
+	// StopGroups group of stops that must be part of the same route.
+	StopGroups *[][]string `json:"stop_groups,omitempty"`
+	// DurationMatrix matrix of durations in seconds between stops.
+	DurationMatrix *[][]float64 `json:"duration_matrix,omitempty"`
+	// DistanceMatrix matrix of distances in meters between stops.
+	DistanceMatrix *[][]float64 `json:"distance_matrix,omitempty"`
+	// DurationGroups duration in seconds added when approaching the group.
 	DurationGroups *[]DurationGroup `json:"duration_groups,omitempty"`
-	Vehicles       []Vehicle        `json:"vehicles,omitempty"`
-	Stops          []Stop           `json:"stops,omitempty"`
+	// Vehicles to route.
+	Vehicles []Vehicle `json:"vehicles,omitempty"`
+	// Stops that will be routed and assigned to the vehicles.
+	Stops []Stop `json:"stops,omitempty"`
+	// AlternateStops a set of alternate stops for vehicles.
```

Comment:
We should explain in some more detail how alternate stops will be handled by the solver. In short. Details are for docs. :relaxed: